<?xml version="1.0"?>
<launch>

  <!-- load the controllers -->
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/arobot" args="arobot_trajectory_controller joint_state_controller" />


  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
	  respawn="false" output="screen">
	  <remap from="/joint_states" to="/arobot/joint_states/" />
  </node>

</launch>

